app/src/main/java/com/sdpteam/connectout/event/viewer/EventActivity.java

```diff
@@ -78,32 +80,48 @@ private void initViewModel() {
     private void initEventView() {
         TextView title = findViewById(R.id.event_title);
         TextView description = findViewById(R.id.event_description);
-        Button participationBtn = findViewById(R.id.event_join_button);
+        Button joinBtn = findViewById(R.id.event_join_button);
+        Button interestedBtn = findViewById(R.id.event_interested_button);
         Button participantsBtn = findViewById(R.id.event_participants_button);
         ImageButton chatBtn = findViewById(R.id.event_chat_btn);
 
-        participationBtn.setOnClickListener(v -> {
-            eventViewModel.toggleParticipation(currentUserId);
-        });
```

Comment:
not sure why you delete this. Anyways, it's an essential part of my pr so I ll see that with you when we meet up for the merge